src/modules/Header/components/Header.js

```diff
@@ -69,8 +70,18 @@ const HeaderBarLower = styled.div`
 `
 
 const HeaderUpperRight = styled.div`
-  padding: 14px 20px;
-  line-height: 18px;
+  display: flex;
+  flex-wrap: wrap;
+  padding-right: 20px;
```

Comment:
We shouldn't use pixels. They don't scale. With predefined paddings (and margins) we can make the UI also more coherent.